lessons/codap-bar-charts/langs/en-us/pages/notes-plotting-and-displaying-data.adoc

```diff
@@ -15,7 +15,7 @@
 
 @vspace{1ex}
 
-Data Scientists use *displays* to visualize data. You've probably seen some of these charts, graphs and plots yourselves! When it comes to displaying *Categorical Data*, we often rely on *bar charts* and *pie charts*. CODAP, however, only has the capacity to create *bar charts*.
+Data Scientists use *displays* to visualize data. You've probably seen some of these charts, graphs and plots yourselves! When it comes to displaying @vocab{Categorical Data}, we often rely on *dot plots* and @vocab{bar charts}. (Pie charts display categorical data, too, but CODAP doesn't have the abilitly to create them.)
```

Comment:
You may want to add a note for the teacher about why some people think Pie Charts are bad by default: https://www.data-to-viz.com/caveat/pie.html

It's not that CODAP lacks the ability to create Pie Charts -- it's that the CODAP team opposes them on principle

Reply:
I was unaware of this! Good perspective, thanks. Will add.

Reply:
Do you think we should add "dot plot" as vocab? I kinda don't? Noticing now how I bolded it up there.